src/components/Tab/TabItem/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+export interface TabItemProps {
+  name: string;
+  active: boolean;
+  icon?: React.ComponentType<any>;
+  onlyIcon?: boolean;
+  onClick: React.MouseEventHandler<HTMLButtonElement>;
+}
+
+const TabItem: React.FC<TabItemProps> = ({
+  name,
+  active,
+  icon: Icon,
+  onlyIcon = false,
+  ...rest
+}: TabItemProps) => {
+  return (
+    <li className="flex-auto mr-3 text-center last:mr-0">
+      <a
+        className={
+          'font-bold py-1 rounded-lg flex place-items-center justify-center leading-normal ' +
+          (active ? 'text-red-600 bg-red-100' : 'text-gray-500')
+        }
+        {...rest}
+        data-toggle="tab"
+        href={`#${name}`}
```

Comment:
I suggest getting a separate prop for the href. To avoid misuse. I can rewrite the className prop if I want, because of the current implementation. We don't want to let this sort of thing happen.